docs/source/config_file.rst

```diff
@@ -333,6 +333,11 @@ a list of import discovery options that may be used
     Specifies the paths to use, after trying the paths from ``MYPYPATH`` environment
     variable.  Useful if you'd like to keep stubs in your repo, along with the config file.
 
+``files`` (string)
+    Specifies the paths which should be checked by mypy if none are given on the command
+    line. Supports recursive file globbing through
+    [the glob library](https://docs.python.org/3/library/glob.html).
```

Comment:
Also this should mention the separators used (comma, colon, semicolon), or whatever we end up deciding in the end.